packages/shared/src/components/profile/AccountDangerZone.tsx

```diff
@@ -1,64 +1,45 @@
-import classNames from 'classnames';
 import React, { ReactElement, ReactNode } from 'react';
-import {
-  Button,
-  ButtonColor,
-  ButtonSize,
-  ButtonVariant,
-} from '../buttons/Button';
+import { DangerZone } from '../widgets/DangerZone';
 
 interface AccountDangerZoneProps {
   onDelete: () => void;
   className?: string;
   children?: ReactNode;
 }
 
+const Important = () => (
+  <>
+    Important: deleting your account is unrecoverable and cannot be undone. Feel
+    free to contact{' '}
+    <a
+      className="text-text-link"
+      href="mailto:support@daily.dev?subject=I have a question about deleting my account"
+      target="_blank"
+      rel="noopener"
```

Comment:
```suggestion
      rel={anchorDefaultRel}
```

Reply:
Oh, cool. Just copy-pasted this block and didn't realize it. Yep much better to use the common one.